src/model/page-config-resolver.js

```diff
@@ -292,6 +335,120 @@ class MicrodataParser {
     this.doc_ = doc;
   }
 
+  /**
+   * Returns true if access is restricted, otherwise false
+   * @param {Node} root A node that is an item of type 'NewsArticle'
+   * @return {?boolean} locked access
+   * @private
+   */
+  discoverLocked_(root) {
+    const ALREADY_SEEN = 'alreadySeenForAccessInfo';
```

Comment:
There's already `ALREADY_SEEN` constant defined at the top of the file. Can we just use that?

Reply:
offline discussion, we'll keep alreadySeen tag depending on what we are searching for.